docs/blog/20240624-Daeraxa-3kStars.md

```diff
@@ -0,0 +1,166 @@
+
+### GitHub sunsets Atom
+
+<img src="./assets/sunsetting2.png" height="200" />
+
+Despite the reassurances that Atom and VS Code could coexist under Microsoft’s umbrella, it became clear that interest in Atom development was decreasing, new features weren't forthcoming, and issues were being left unresolved. The Atom.io blog page saw its last update in 2019 with the announcement of Atom 1.39; most releases after that featured fixes and updates, but little in the way of exciting new features.
+
+The old Atom forum was shut down (but is still browsable via the [Wayback Machine](https://web.archive.org/web/20210622052601/https://discuss.atom.io/) and moved to the then-new [GitHub Discussions](https://github.com/atom/atom/discussions)… but there was little interaction from the original developers. The original package website languished and didn't have its "Featured" or "Trending" packages updated for a long time.
+
+Finally, on the 8th of June, 2022, GitHub made an announcement that Atom would be [sunset](https://github.blog/2022-06-08-sunsetting-atom/) in favour of VS Code and gave six months’ warning before the various services that make up the Atom ecosystem would go offline forever. The software itself and its source code would, of course, still be available via the archived repository; but the website, Flight Manual, package backend, and Teletype service would no longer be available.
+
+### Atom Community
+
+<img src="./assets/atom-community.png" height="200" />
+
+As soon as the news hit, there was a flurry of activity. Some were looking for direct replacements for Atom; others were looking for active forks or continuations of the project. [Atom Community](https://atom-community.github.io/) was the logical place for these discussions. This was a community that had taken on the maintenance of a bunch of Atom packages (originally for the abandoned Facebook [Nuclide](https://nuclide.io/) project to turn Atom into an IDE) and had since assumed stewardship of other packages from the original Atom maintainers.
+
+At this time, the Atom Community Discord server was rather active with discussions about what could be done. Could the community keep Atom alive? Could they take ownership of any of the original repos? Would a community project be allowed to use the original name? However, it became clear there was somewhat of a split in the goals of the group. Some wanted to take Atom and radically change it, upgrade it and take it into the future — bringing it into the state it might have been in had development not slowed to a crawl. Others wanted to be a little more conservative: keep the Atom they knew and loved and make sure it was kept up-to-date with patches and fixes.
```

Comment:
Maybe this can capture the historical moment of the split with more accuracy?

```suggestion
At this time, the Atom Community Discord server was rather active with discussions about what could be done. Could the community keep Atom alive? Could they take ownership of any of the original repos? Would a community project be allowed to use the original name?

However, amongst the people who gathered there to discuss the future of Atom, there were some differences of vision for the future. Some folks who were still around from the early days of the now two-years-long effort at the atom-ide-community/atom (now atom-community/atom) fork wished to pursue ideas from when that fork was created -- ideas such as moving more JavaScript code to TypeScript, and bundling more IDE features and a terminal package. Whereas in more recent circulation was a roadmap forward from official Atom's seemingly abandoned status, which had gained some enthusiasm from various would-be contributors to a forked Atom -- with goals such as updating to newer Electron, and replacing lost functionality of the atom.io website (which members of the community correctly inferred might get shut down soon along with the overall sunset of Atom). The contributors working toward this roadmap also converged on a desire to revamp and simplify the build process of the editor, along with re-vamping the project's CI, so as to make contribution more manageable. And to make some bolder changes/fixes, including for developer quality of life when revamping/modernizing/maintaining the code, which might not immediately pass in the previously existing CI.
```

Reply:
A bit verbose, gonna suggest something shorter. Marking this draft/suggestion as resolved.